### PR TITLE
Migrate capseuratconverter to log4r

### PR DESCRIPTION
**Pull request by siberianisaev**
- log4r usage
- service version update

### CODE REVIEW

File: R/logs.R [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+log_file_name <- read_environment_variable(
+  "CAP_SEURAT_CONVERTER_LOG_FILE_NAME",
+  paste(getwd(), "logfile.txt",
+    sep = "/src/"
+  )
+)
+log_threshold <- read_environment_variable(
+  "CAP_SEURAT_CONVERTER_LOG_LEVEL",
+  "DEBUG"
+)
```
Comment (rm1113) [inline R/logs.R:20]: Don't need to do that in the basic package, we would control in on the server side

File: R/logs.R [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  log4r::error(log, msg)
+}
+
+log_debug <- function(msg) {
+  log4r::debug(log, msg)
+}
+
+log_warn <- function(msg) {
+  log4r::warn(log, msg)
+}
```
Comment (rm1113) [inline R/logs.R:47]: Let's move it to the main file to simplify the package structure

**Review by rm1113 — CHANGES_REQUESTED** [review]

File: R/logs.R (hunk 1 above)
Comment (siberianisaev) [reply R/logs.R]: done

File: R/logs.R (hunk 2 above)
Comment (siberianisaev) [reply R/logs.R]: ready

**Review by rm1113 — APPROVED** [review]

**Pull request merged**